apps/settings/js/call.js

```diff
@@ -1,3 +1,5 @@
+/* global getIccByIndex, SettingsHelper */
+/* jshint -W083 */
```

Comment:
for call.js, I don't want to do too much works here because I will totally removed all of them at my next patch that you reviewed before. So, let's keep this file intact, ok ?